src/components/views/rooms/EventTile.tsx

```diff
@@ -1032,6 +1032,13 @@ export class UnwrappedEventTile extends React.Component<IProps, IState> {
         if (this.context.timelineRenderingType === TimelineRenderingType.Notification) {
             avatarSize = 24;
             needsSenderProfile = true;
+        } else if (isInfoMessage || isInfoMessage &&
+            (this.context.timelineRenderingType === TimelineRenderingType.Thread && !this.props.continuation)
+        ) {
```

Comment:
This expression might be wrong given that it simplifies to just `isInfoMessage` ignoring the whole second part (`&&` has precedence over `||`)

```suggestion
        } else if (isInfoMessage || isInfoMessage &&
            (this.context.timelineRenderingType === TimelineRenderingType.Thread && !this.props.continuation)
        ) {
```

Reply:
@t3chguy Would you please add suggestion for me as I am not really sure what the best solution would be? Thanks in advance!